akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +58,11 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    CompletionStage<Done> ignoreDataBytesJ(Materializer materializer);
```

Comment:
> Speaking of which, dataBytes is a name which means virtually anything/nothing.

Why? Normally one accesses it via: `response.entity.dataBytes`, I do think that's rather readable / self-explaining.

> Assuming that name is not forever set in stone, how about discardDataBytes?
 
That's an interesting idea actually.
To remind context of why we're discussing: the same entity will expose both scala/java APIs, so we'd need `discardDataBytes: CompletionStage...` and `ignoreDataBytes: Future...` (also not happy...) or something else.

I also had an idea of `ignoreDataBytes().future` / `ignoreDataBytes().completionStage`?